### DISCUSSION TITLE
How to add Hyperlink on the window, I can't find it in docs.

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#13309] [Q&A] [answered]
Like `<a href="https://github.com/">go github</a>`, click and the default browser will open.

**Comment by maxkatz6** [comment, marked as answer]
It's not a web page, so it's not as straightforward. 
https://github.com/AvaloniaUI/Avalonia/issues/7640

**Reply by IronEaterBeast** [reply]
Ok, got it, thank you.
